contracts/Compliance.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity 0.5.16;
+
+import "@openzeppelin/contracts/ownership/Ownable.sol";
+import "./spec_interfaces/ICompliance.sol";
+
+contract Compliance is ICompliance, Ownable {
+
+    IContractRegistry contractRegistry;
+
+    mapping (address => string) validatorCompliance;
+
+    /*
+     * External methods
+     */
+
+    function getValidatorCompliance(address addr) external view returns (string memory conformanceType) {
+        string memory compliance = validatorCompliance[addr];
+        if (bytes(compliance).length == 0) {
+            compliance = "General"; // TODO should this be configurable?
```

Comment:
The word compliance imply multiple string keys mapping to boolean values. such as: `compliance[addr][identified] == true`.

If we want to be specific, lets not call it compliance, it can be ValidatorIdentification contract. and the value can be boolean. If we are generic lets be generic because compliance is not a single string value you can comply to different things: GDPR compliant for instance, etc.. 

in other words, if you are "Identified" it does not mean you are not "General"...


@OdedWx, @vistra  ?

Reply:
Sounds good to me, I'll add a TODO for us to decide later. I prefer not to block the merge over this.